source/codegen/metadata/niscope/functions.py

```diff
@@ -1440,7 +1366,7 @@
                 'name': 'errorSource',
                 'size': {
                     'mechanism': 'fixed',
-                    'value': 642
+                    'value': 55
```

Comment:
This is interesting that this changed. Needs further investigation.

Reply:
So i think this one is actually a correct change 642 is the max error description size, and 55 is the max function name size so therefore errorSource which is the parameter telling us the function source seems like 55 is more correct. It is strange that it didn't change in the service though